hydra/.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
Since specs can now be run, it would be helpful to have something about running specs added into the development.rb documentation.

Reply:
Cool, I added some basic documentation on running specs